frontend/README.md

```diff
@@ -0,0 +1,71 @@
+# Shipment Tracker frontend
+
+This is a React and Typescript application bootstrapped using [Create React App](https://create-react-app.dev/)
+
+## Development setup
+
+1. **Install the dependencies CLI**
+
+   ```shell
+   yarn install
+
+   ```
+
+2. **Start the app**
+
+   Get your Gatsby blog set up in a single command:
```

Comment:
Gatsby?

Reply:
🤦 that's what happens when you copy-paste from nice readmes